keep/conditions/threshold_condition.py

```diff
@@ -10,19 +10,56 @@ class ThresholdCondition(BaseCondition):
         BaseCondition (_type_): _description_
     """
 
-    def __init__(self, condition_type, condition_config):
-        super().__init__(condition_type, condition_config)
+    def __init__(self, *kargs, **kwargs):
+        super().__init__(*kargs, **kwargs)
+        self.levels = []
 
-    def apply(self, compare_to, compare_value) -> bool:
-        """apply the condition.
+    def _check_if_multithreshold(self, compare_to):
+        """Checks if this is a multithreshold condition.
+
+        Args:
+            compare_to (str): for single threshold could be 60 or 60%, for multithreshold
+                              will be 60, 70, 80 (comma separated values)
+
+        Raises:
+            ValueError: If the number of levels and number of thresholds do not match
+
+        Returns:
+            bool: True if multithreshold, False otherwise
+        """
+        # TODO make more validations
+        if "," in str(compare_to):
+            levels = self.condition_config.get("level")
+            if len(levels.split(", ")) != len(compare_to.split(",")):
+                raise ValueError(
+                    "Number of levels and number of thresholds do not match"
+                )
+            self.levels = [l.strip() for l in levels.split(",")]
+            return True
+        return False
+
+    def _apply_multithreshold(self, compare_to, compare_value):
```

Comment:
side note: we have to change "compare_to" and "compare_value", it makes no sense